src/Microsoft.ML.SearchSpace/ICanSummarize.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.ML.SearchSpace
+{
+    internal class Schema
```

Comment:
Could we change the name of this class from Schema to something else? We already use schema for many things, and this seems more like a summary rather than a schema.